doc/project/release_process.rst

```diff
@@ -277,7 +408,7 @@ Each release is created with the above products to document the quality and the
 state of the software when it was released.
 
 Long Term Support and Maintenance
-++++++++++++++++++++++++++++++++++
+---------------------------------
 
 A Zephyr LTS release is published every 2 years and is branched and maintained
```

Comment:
I've had some feedback that says that the support period should actually be longer. The main reason being that it often takes 2 years just to get a product to market, and some product makers want longer-term stability.

On the other hand, Nordic, who bases the NRF Connect SDK on Zephyr needs the stability but also needs the feature updates on a regular basis. It's a bit of a double-edged sword.

Reply:
this is orthogonal, the LTS timeline and cadence it s a different topic to be changed sperately. I am not changing this text in this PR.